missing Documenter.jl dependency during build?

new error in building/installing 1.5.1 on ubuntu:
```
$ make install
    CC usr/tools/stringreplace
make[1]: Entering directory '/root/julia'
make[2]: Entering directory '/root/julia/stdlib'
make[2]: Leaving directory '/root/julia/stdlib'
make[2]: Entering directory '/root/julia/deps'
make[2]: Leaving directory '/root/julia/deps'
make[2]: Entering directory '/root/julia/base'
make[2]: Leaving directory '/root/julia/base'
make[2]: Entering directory '/root/julia/src'
make[2]: Leaving directory '/root/julia/src'
make[2]: Entering directory '/root/julia/src'
make[2]: Leaving directory '/root/julia/src'
make[2]: Entering directory '/root/julia/ui'
make[2]: Leaving directory '/root/julia/ui'
make[2]: Entering directory '/root/julia'
make[2]: Leaving directory '/root/julia'
make[2]: Entering directory '/root/julia'
make[2]: Leaving directory '/root/julia'
make[2]: Entering directory '/root/julia/doc'
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   447  100   447    0     0   2829      0 --:--:-- --:--:-- --:--:--  2829
100 1808k  100 1808k    0     0  6171k      0 --:--:-- --:--:-- --:--:-- 6171k
Building HTML documentation.
 Installing known registries into `~/julia/doc/deps`
######################################################################## 100.0%
      Added registry `General` to `~/julia/doc/deps/registries/General`
   Updating registry at `~/julia/doc/deps/registries/General`
No Changes to `~/.julia/environments/v1.5.1/Project.toml`
No Changes to `~/.julia/environments/v1.5.1/Manifest.toml`
ERROR: LoadError: ArgumentError: Package Documenter [e30172f5-a6a5-5a46-863b-614d45cd2de4] is required but does not seem to be installed:
 - Run `Pkg.instantiate()` to install all recorded dependencies.

Stacktrace:
 [1] _require(::Base.PkgId) at ./loading.jl:999
 [2] require(::Base.PkgId) at ./loading.jl:928
 [3] require(::Module, ::Symbol) at ./loading.jl:923
 [4] include(::Function, ::Module, ::String) at ./Base.jl:380
 [5] include(::Module, ::String) at ./Base.jl:368
 [6] exec_options(::Base.JLOptions) at ./client.jl:296
 [7] _start() at ./client.jl:506
in expression starting at /root/julia/doc/make.jl:9
make[2]: *** [html] Error 1
make[1]: *** [docs] Error 2
make: *** [/root/julia/doc/_build/html/en/index.html] Error 2
```

--- a/doc/make.jl
+++ b/doc/make.jl
@@ -1,4 +1,5 @@
 # Install dependencies needed to build the documentation.
+Base.ACTIVE_PROJECT[] = nothing
 empty!(LOAD_PATH)
 push!(LOAD_PATH, @__DIR__, "@stdlib")
 empty!(DEPOT_PATH)

